db/migrate/20150720142931_reregister_archived_whitehall_artefact_routes.rb

```diff
@@ -0,0 +1,14 @@
+class ReregisterArchivedWhitehallArtefactRoutes < Mongoid::Migration
+  def self.up
+    Artefact.where(:state => 'archived', :owning_app => 'whitehall').each do |artefact|
+      # Set state to live (but don't save) so that RoutableArtefact creates the
+      # missing routes for these.
+      artefact.state = 'live'
+      RoutableArtefact.new(artefact).submit(:skip_commit => true)
```

Comment:
Fair point.  I'd assumed (incorrectly) that `register` was a private method.  This does make me wonder whether it should be, but that's something for another PR.

Updated and force-pushed.